src/Components/Facility/FacilityCard.tsx

```diff
@@ -95,7 +95,7 @@ export const FacilityCard = (props: { facility: any; userType: any }) => {
                       {facility.name}
                     </Link>
                     <ButtonV2
-                      href={`/facility/${facility.id}/cns`}
+                      href={`/facility/${facility.id}/cns?page=1`}
```

Comment:
@patelaryan7751 
instead of hardcoding the page, why not make the page = 1 when there is no query params passed

Reply:
Hey @khavinshankar i have removed all hardcoded links